blog/_posts/2016-10-12-Hackbright.md

```diff
@@ -0,0 +1,123 @@
+---
+ title:
```

Comment:
you need a title for the post — best would be to move the text on line 12 to here :)